server/disk_avail.go

```diff
@@ -11,8 +11,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build !windows && !openbsd && !wasm
-// +build !windows,!openbsd,!wasm
+//go:build !windows && !openbsd && !netbsd && !wasm
```

Comment:
Since you modified this file, could you please add `-2022` to the very first line of this file (the copyright banner)?